src/dali/plugin/input/rest/binance_com.py

```diff
@@ -415,49 +426,51 @@ def _process_gains(self, in_transactions: List[InTransaction]) -> None:
                 #     "accountProfits": [
                 #       {
                 #         "time": 1586188800000,            // Mining date
-                #         "type": 31, // 0:Mining Wallet,5:Mining Address,7:Pool Savings,
+                #         "type": "31", // 0:Mining Wallet,5:Mining Address,7:Pool Savings,
                 #           8:Transferred,31:Income Transfer ,32:Hashrate Resale-Mining Wallet 33:Hashrate Resale-Pool Savings
                 #         "hashTransfer": null,            // Transferred Hashrate
                 #         "transferAmount": null,          // Transferred Income
                 #         "dayHashRate": 129129903378244,  // Daily Hashrate
                 #         "profitAmount": 8.6083060304,   //Earnings Amount
                 #         "coinName":"BTC",              // Coin Type
-                #         "status": 2    //Status：0:Unpaid， 1:Paying  2：Paid
+                #         "status": "2"    //Status：0:Unpaid， 1:Paying  2：Paid
                 #       },
                 #       {
                 #         "time": 1607529600000,
                 #         "coinName": "BTC",
-                #         "type": 0,
+                #         "type": "0", // String
                 #         "dayHashRate": 9942053925926,
                 #         "profitAmount": 0.85426469,
                 #         "hashTransfer": 200000000000,
                 #         "transferAmount": 0.02180958,
-                #         "status": 2
+                #         "status": "2"
                 #       },
                 #       {
                 #         "time": 1607443200000,
                 #         "coinName": "BTC",
-                #         "type": 31,
+                #         "type": "31",
                 #         "dayHashRate": 200000000000,
                 #         "profitAmount": 0.02905916,
                 #         "hashTransfer": null,
                 #         "transferAmount": null,
-                #         "status": 2
+                #         "status": "2"
                 #       }
                 #     ],
                 #     "totalNum": 3,          // Total Rows
                 #     "pageSize": 20          // Rows per page
                 #   }
                 # }
 
-                if _DATA in results:
+                if results[_DATA][_TOTALNUM] != "0":
+                    # For testing - REMOVE

```

Comment:
If it's something to remove after testing, mark it as TODO (pylint will pick it up and remind you):
```suggestion
                    # TODO: For testing - REMOVE
```

Reply:
This was just an extra logger I needed to inject to figure out what was going wrong.